velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -544,6 +544,11 @@ struct MinMaxNAccumulator {
     VELOX_USER_CHECK_GT(
         newN, 0, "second argument of max/min must be a positive integer");
 
+    VELOX_USER_CHECK_LE(
+        newN,
+        10'000,
+        "second argument of max_n/min_n must be less than or equal to 10000");
```

Comment:
Let's match the previous error message: max_n/min_n -> max/min.

I did notice that this is the message from Presto, but it seems wrong to refer to non-existent functions min_x and max_n.

Reply:
Fixed